docs/TSCONFIG.md

```diff
@@ -0,0 +1,150 @@
+# TsConfig files structure
+
+The presence of a config file in a directory indicates that it is the root of a TypeScript project.
+In Stark, several types of tsconfig files are placed in the packages for different purposes, described below.
+
+## tsconfig.json
+
+The `tsconfig.json` file can be found at the following locations:
+
+```txt
++---starter
+|	tsconfig.json
++---showcase
+|	tsconfig.json
++---packages
+	tsconfig.json
+	+---stark-build
+	|	tsconfig.json
```

Comment:
The other packages are missing here

Reply:
Thanks for your comments Christopher, I indeed needed help to complete that doc :) I come back to you in a bit!

Reply:
Ups, indeed, my bad :p